recursion/recursionFibonacci/FibonacciPrinter.go

```diff
@@ -0,0 +1,5 @@
+package FibonacciElements
+
+type FibonacciPrinter interface {
+	recursionFibonacci(n int) int
```

Comment:
use name like calcFibonacci(recur bool) here if we call with true then it computes using recursion otherwise not. Or you can use iota to pass in the type of computation if there are more than 2